data/json/npcs/exodii/exodii_weaponmaster_talk.json

```
@@ -0,0 +1,76 @@
+[
+  {
+    "id": "TALK_WEAPONMASTER_INTRO_1",
+    "type": "talk_topic",
+    "//": "mute by choise, and deaf most of the time (don't like to turn on the hearing cbm unless for job)",
+    "dynamic_line": "&What you see looks like the back of a mechanical spider, with at least a dozen crude but polished metal arms going from the enormously wide spine and shoulders.  Spine of a creature, meanwhile, sits in an awkward-looking chair, doing something on a wide, heavily lighted workbench.  Creature seems to be busy with something.",
+    "responses": [
+      { "text": "&Wait a bit.", "effect": "start_trade", "topic": "TALK_WEAPONMASTER_INTRO_2A" },
```

Narrative responses use a style like this:
https://github.com/CleverRaven/Cataclysm-DDA/blob/5d320508149b28da05fc4d45e4a8f2d1c7547ef6/data/json/npcs/robofac/robofac_intercom/robofac_intercom_missions.json#L592
```suggestion
      { "text": "[Wait a bit.]", "effect": "start_trade", "topic": "TALK_WEAPONMASTER_INTRO_2A" },
```